src/Aquifer.Common/Messages/QueueClientFactory.cs

```diff
@@ -52,13 +53,15 @@ public async Task<QueueClient> GetQueueClientAsync(string queueName, Cancellatio
     // Note: This method creates the queue if it doesn't yet exist.
     private async Task<QueueClient> GetQueueClientCoreAsync(string queueName, CancellationToken ct)
     {
-        var client = _queueConfigurationOptions.AzureQueueStorageConnectionString.StartsWith("http")
+        var client = string.IsNullOrEmpty(_azureStorageAccountOptions.ConnectionStringOverride)
             ? new QueueClient(
-                new Uri($"{_queueConfigurationOptions.AzureQueueStorageConnectionString}/{queueName}"),
```

Comment:
This was a regression.  Fixed here: https://github.com/BiblioNexusStudio/aquifer-server/pull/711